ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -364,8 +370,14 @@ impl Blockstore {
 
         let mut index0 = self.transaction_status_index_cf.get(0)?.unwrap_or_default();
         let mut index1 = self.transaction_status_index_cf.get(1)?.unwrap_or_default();
+        let r_highest_primary_index_slot = self.highest_primary_index_slot.read().unwrap();
+        let highest_primary_index_slot = *r_highest_primary_index_slot;
+        drop(r_highest_primary_index_slot);
```

Comment:
```suggestion
        let highest_primary_index_slot = self.get_highest_primary_index_slot();
```
nit: use the existing getter

Reply:
Oh thanks! I forgot to fixup after I added the getter